pkg/sql/codegen/xgboost/codegen.go

```diff
@@ -37,6 +37,7 @@ range: [0,1]`, attribute.Float32RangeChecker(0, 1, true, true)},
 range: [2, Infinity]`, attribute.IntLowerBoundChecker(2, true)},
 	"objective":   {attribute.String, nil, `Learning objective`, objectiveChecker},
 	"eval_metric": {attribute.String, nil, `eval metric`, nil},
+	"train.cache": {attribute.Bool, false, `whether use external memory to cache train data`, nil},
```

Comment:
Seems the word cache stands for caching things in memory generally. Maybe the option name should be like `use_swap` etc.

Reply:
Done.